scripts/export-for-ops.ts

```diff
@@ -142,54 +163,53 @@ const main = async () => {
-
-      const hdNode = verifyAirnodeXpub(apiMetadata.xpub, templateObj.airnode);
-      const keeperSponsorPath = deriveKeeperWalletPathFromSponsorAddress(keeperSponsor);
-      const apiProviderUpdateRequester = hdNode.derivePath(keeperSponsorPath).address;
-
-      const api3HdNode = ethers.utils.HDNode.fromExtendedKey(api3Xpub);
-      const api3AirnodeAddress = api3HdNode.derivePath('0/0').address;
-      const encodedParameters = abi.encode(parameters);
-      const beaconId = ethers.utils.solidityKeccak256(['bytes32', 'bytes'], [templateId, encodedParameters]);
-      const beaconDescriptor = {
+    jobs.map(
+      async ({
         templateId,
-        parameters: templateObj.parameters,
-        decodedParameters: templateObj.decodedParameters,
-        beaconId: beaconId,
-        chains: await Promise.all(
-          templateObj.chains.map(async (chainName: string): Promise<ChainDescriptor> => {
-            return {
-              name: chainName,
-              apiProviderAirkeeperDeviationPercentage: parseFloat(deviationPercentage),
-              api3AirkeeperDeviationPercentage: parseFloat(deviationPercentage) * 2,
-              apiProviderUpdateRequesterMetadata: { address: apiProviderUpdateRequester },
-              apiProviderAirnodeFulfillerMetadata: {
-                address: await deriveSponsorWalletAddress(apiMetadata.xpub, templateObj.airnode, requestSponsor),
-              },
-              api3ProviderUpdateRequesterMetadata: {
-                address: await deriveSponsorWalletAddress(api3Xpub, api3AirnodeAddress, requestSponsor),
-              },
-            };
-          })
-        ),
-      };
-
-      await writeBeaconDescriptor(
-        join(beaconsBasePath, `${templateId}-${sanitiseFilename(endpointName)}.json`),
-        beaconDescriptor
-      );
-      return beaconDescriptor;
-    })
+        templateParameters,
+        overrideParameters,
+        deviationPercentage,
+        keeperSponsor,
+        requestSponsor,
+      }) => {
+        const templateObj = templates.find((template) => template.templateId === templateId);
+
+        const encodedParameters = abi.encode([...templateParameters, ...overrideParameters]);
+        const beaconId = ethers.utils.solidityKeccak256(['bytes32', 'bytes'], [templateId, encodedParameters]);
+        const beaconDescriptor = {
+          templateId,
+          templateName: templateObj.name,
+          parameters: templateObj.parameters,
+          decodedParameters: templateObj.decodedParameters,
+          beaconId: beaconId,
+          chains: await Promise.all(
+            templateObj.chains.map(async (chainName: string): Promise<ChainDescriptor> => {
+              return {
+                name: chainName,
+                sponsor: requestSponsor,
+                apiProviderAirkeeperDeviationPercentage: parseFloat(deviationPercentage),
+                api3AirkeeperDeviationPercentage: parseFloat(deviationPercentage) * 2,
```

Comment:
Just wondering, what would be the process to deploy an API3 operated beacon? I'm assuming we won't be using this guide for that but we will be reading this value...

Reply:
I agree, I'll remove that value (and if it isn't present in the consuming apps apply a default).